### ISSUE TITLE
Accessibility: Audit My Site fragment and nested screens (excluding Stats and Editor)

### ISSUE THREAD

**Issue by khaykov** [wordpress-mobile/WordPress-Android#7283]
Audit My Site fragment and nested screens (excluding Stats and Editor) to make sure that all their functionality is accessible with TalkBack and Switch Access.

- All the clickable elements should be marked as so.
- All nonactionable and decorative elements should be non-focusable.
- Relevant content should be grouped together.
- All important elements should be described through content description.
